specification.markdown

```diff
@@ -556,13 +556,13 @@ minOccurs: 1
 
 maxOccurs: 1
 
-range:  1
+range:  not 0
```

Comment:
As noted on Cellar, only version 1 is defined, so any other value would be an undefined implementation. I suggest to leave as `range: 1` and when/if an EBML specification of version 2 is made then this is changed to `range: 1-2`.